src/mongoose_cluster_id.erl

```diff
@@ -96,20 +110,8 @@ store_cluster_id(ID) ->
 
 -spec set_new_cluster_id(cluster_id(), mongoose_backend()) -> ok | {error, any()}.
 set_new_cluster_id(ID, rdbms) ->
-    SQLQuery = [<<"INSERT INTO mongoose_cluster_id(k,v) "
-                  "VALUES ('cluster_id',">>,
-                mongoose_rdbms:use_escaped(mongoose_rdbms:escape_string(ID)), ");"],
-    try mongoose_rdbms:sql_query(global, SQLQuery) of
-        {updated, 1} -> {ok, ID};
-        {error, _} = Err -> Err
-    catch
-        Class:Reason:Stacktrace ->
-            ?LOG_WARNING(#{what => cluster_id_set_failed,
-                           text => <<"Error inserting cluster ID into RDBMS">>,
-                           sql_query => SQLQuery, cluster_id => ID,
-                           class => Class, reason => Reason, stacktrace => Stacktrace}),
-            {error, {Class, Reason}}
-    end;
+    Res = execute_cluster_insert_new(ID),
+    convert_insert_response(Res, ID);
```

Comment:
I think we need to keep the try...catch logic here as the code intentionally falls back to Mnesia if RDBMS is faulty to prevent a crash.

Reply:
Update: we checked that MongooseIM crashes when the DB is not available anyway, so no need for try...catch.